lib/loggerator/metrics.rb

```diff
@@ -1,18 +1,18 @@
-require_relative 'loggerator'
+require "loggerator"
 
 module Loggerator
   module Metrics
     include Loggerator
     extend self
 
-    @@metrics_name = 'loggerator'
+    Loggerator.config.metrics_app_name ||= "loggerator"
 
     def name=(name)
-      @@metrics_name = name
+      Loggerator.config.metrics_app_name = name
     end
 
     def name
-      @@metrics_name
+      Loggerator.config.metrics_app_name
     end
```

Comment:
Gotcha. I could also set `"loggerator"` in the `Configuration#initialize` method.